lib/src/keypair.dart

```diff
@@ -5,7 +5,7 @@ abstract class Keypair {
   Keypair(PrivateKey privateKey);
 
   /// Generate a random [Keypair]
-  Keypair.fromRandom();
+  Keypair.fromRandom({int keySize});
```

Comment:
```suggestion
  Keypair.fromRandom();
```
Undo, because it doesn't affect the Elliptic curve